packages/secretlint/test/create-secretlintrc.test.ts

```diff
@@ -0,0 +1,67 @@
+// LICENSE : MIT
+"use strict";
+import assert from "assert";
+import path from "path";
+import fs from "fs";
+import os from "os";
+import { runConfigCreator } from "../src/create-secretlintrc";
+
+/**
+ *
+ * Create .secretlintrc.json
+ * - yarn test -g "create-secretlintrc-test"
+ *
+ * File Overview
+ *
+ * - tmpConfigDir
+ *   - A temporal directory with "secretlint-config"
+ *   - Create .secretlintrc.json on the directory
+ * - packageFilePath
+ *   - A path of package.json to use of this test
+ *   - When using this package.json, copy it to tmpConfigDir
+ * - expectedConfigFilePath
+ *   - A path of .secretlintrc.json to create by this test
+ */
+describe("create-secretlintrc-test", function () {
```

Comment:
```suggestion
describe("create-secretlintrc", function () {
```

This is a very minor detail, but "test" is unnecessary because it duplicates the meaning.